src/plugins/blocksmith/components/instagram-media-block-modal/index.test.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import test from 'tape';
+import INSTAGRAM_MEDIA_REGEX from './instagramMediaBlockRegex';
+
+test('InstagramMediaBlock', (t) => {
```

Comment:
@jalopez13 this isnt testing index so it should not be called `index.test.jsx`. name it after the file it is testing

Reply:
@efjacobson 

Renamed to `instagramMediaBlockRegex.test.js`.